samples/zigbee/light_bulb/README.rst

```diff
@@ -0,0 +1,108 @@
+.. _zigbee_light_bulb_sample:
+
+Zigbee: Light bulb
+##################
+
+This Zigbee light bulb sample demonstrates a simple light bulb whose brightness can be regulated by a different device.
+
+You can use this sample together with :ref:`Zigbee network coordinator <zigbee_network_coordinator_sample>` and Zigbee light switch to set up a basic Zigbee network.
+
+Overview
+********
+
+The Zigbee light bulb sample assumes the Zigbee Router role and implements the Dimmable Light profile.
+This profile allows to change the brightness level of a LED on the light bulb development kit.
+
+In the default sample configuration, the changes to the light bulb brightness are reflected on LED 4.
+
+Requirements
+************
+
+* One or more of the following development kits:
+
+  * |nRF52840DK|
+  * |nRF52840Dongle|
+  * |nRF52833DK|
+
+* The :ref:`Zigbee network coordinator <zigbee_network_coordinator_sample>` application programmed on one separate device.
+* The Zigbee light switch application programmed on one or more separate devices.
+
+You can mix different development kits.
+
+User interface
+**************
+
+LED 1 and LED 2:
+    Indicates the |BLE| status:
+
+    * Blinking - |BLE| advertising ongoing.
+    * Slow blinking (period of 200 ms when on, period of 800 ms when off) - Device is not connecting to a Zigbee mesh network.
+    * Rapid blinking (period of 100 ms) - Device is connecting to a Zigbee mesh network.
+    * Solid - Device is connected over |BLE|.
+
+LED 3:
+    Indicates whether the network is open or closed.
+
+    .. note::
+         When you use the light bulb on the |nRF52840Dongle|, it is the LED 3 that informs about the successful network joining.
+
+LED 4:
+    Indicates the dimmable light option.
+
+Button 4:
+    Puts the light bulb in the Identify mode.
+
+Building and running
+********************
+.. |sample path| replace:: :file:`samples/zigbee/light_bulb`
+
+|enable_zigbee_before_testing|
+
+.. include:: /includes/build_and_run.txt
+
+.. _zigbee_light_bulb_sample_testing:
+
+Testing
+=======
+
+After programming the sample to your development kits, test it by performing the following steps:
+
+1. Turn on the coordinator development kit.
+   When LED 3 turns on, this development kit has become the coordinator of the Zigbee network.
+#. Turn on the light bulb development kit.
+   When LED 3 turns on on the light bulb development kit, it has become a Router inside the network.
+
+   .. tip::
+        If LED 3 does not turn on, press Button 1 on the coordinator to reopen the network.
+
+#. Turn on the light switch development kit.
```

Comment:
same again :)

Reply:
All occurrences of running the sample on DK fixed.